team.md

```diff
@@ -27,7 +27,7 @@ of Cloud Haskell as a whole. [Edsko De Vries][13], a member of Well-Typed and th
 author of much of the new implementation we have today, is still closely involved
 as well.
 
-[Tim][6] is the primary author and maintainer of [disributed-process-platform][8];
+[Tim][6] is the primary author and maintainer of [disributed-process][8];
```

Comment:
Let's remove this part, firstly it's not true (I'm only one of several authors and committers, and distributed-process is now under @facundominguez's maintainership), secondly we're not listing all the libraries. I think we just s/[distributed-process][0],numerous libraries in (... an effort to...)/

thoughts...?